dynawo/3rdParty/sundials/patch/common/sundials.patch

```diff
@@ -154,13 +154,26 @@ index 5c1868f..0186b4b 100644
 +  return (IDA_SUCCESS);
 +}
 +
-+/*----------------------------END IDASetMinStep-------------------------------------*/
```

Comment:
It is here on purpose to trick patch to avoid multiple apply of the patch, if not patch is too smart for his on good and shift lines to apply it anyway

Reply:
Solved